shuup/core/basket/objects.py

```diff
@@ -178,6 +178,8 @@ def clear_all(self):
         self._data = {}
         self.uncache()
         self.dirty = True
+        self.shipping_method = None
```

Comment:
Yes I know, it was kind of my intention to really save that. Dunno if it's okay though